presto-hive/src/main/java/io/prestosql/plugin/hive/rubix/RubixInitializer.java

```diff
@@ -104,6 +108,12 @@ public RubixInitializer(
     @PostConstruct
     public void initializeRubix()
     {
+        if (nodeManager.getCurrentNode().isCoordinator() && !startServerOnCoordinator) {
+            // setup JMX metrics on master (instead of starting server) so that JMX connector can be used
+            setupRubixMetrics();
```

Comment:
I think it will be more readable if this was called from `startRubix`. Technically it is also starting of rubix services. We just start a subset of those on coordinator.

Reply:
> I think it will be more readable if this was called from startRubix

We don't start server (no sockets are opened), we just set it up and register metrics.